estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# licence
+
+from odoo import api
+from odoo import fields
+from odoo import models
+
+
+class EstatePropertyOffer(models.Model):
+    _name = 'estate.property.offer'
+    _description = "Property offers"
+
+    name = fields.Char("Name", required=False, default="- no name -")
+    price = fields.Float("Offer price", required=True)
+    validity = fields.Integer("Validity time", default=7)
+    # Reserved
+    state = fields.Selection(
+        string="Status",
+        selection=[
+            ('accepted', "Accepted"),
+            ('refused', "Refused")
+            ],
+        default=None,
+        )
```

Comment:
```suggestion
    state = fields.Selection(
        string="Status",
        selection=[
            ('accepted', "Accepted"),
            ('refused', "Refused"),
        ],
        default=None,
    )
```
same reason as before